### ISSUE TITLE
tray icon not working on Ubuntu 22.04

### ISSUE THREAD

**Issue by shmu26** [flathub/com.dropbox.Client#265]
Neither right click nor left click displays menu.
The deb version of the app displays menus properly, but has a different issue:
https://bugs.launchpad.net/ubuntu/+source/nautilus-dropbox/+bug/1971558?comments=all

**Comment by wjt** [comment]
True. If you double- or treble-click it it will normally start working mostly OK for the rest of the session.

**Comment by kaimast** [comment]
> True. If you double- or treble-click it it will normally start working mostly OK for the rest of the session.

That's not working for me. I tried on both Wayland and X11.

Weirdly, I do not have this issue on Arch.

**Comment by RodneyBrown** [comment]
Same here, tray icon shows but does nothing.  Same issue on Fedora 36 and Fedora 36 Silverblue

**Comment by nvergottini** [comment]
I'm having the same issue on Fedora 37 Silverblue with Wayland. If I click on the Dropbox tray icon, I don't get the menu, but if I then double-click on the tray icon, the menu shows up in the middle of the screen, but it is not styled correctly for Gnome. Interestingly, if I then disable/enable the "AppIndicator and KStatusNotifierItem Support" Gnome extension, the menu starts to work properly (one click on the tray icon, in the proper location, and styled correctly).

**Comment by apantsiop** [comment]
Ubuntu 22.04. Same here.

**Comment by vitaminace33** [comment]
For me it is working fine on Ubuntu 22.04, but not responding on Manjaro.

**Comment by voedev** [comment]
Fedora 38 KDE and Fedora 38 Gnome

The tray always displays the "Connected" status :(

![Screenshot_20230714_095142](https://github.com/flathub/com.dropbox.Client/assets/42519520/e5433410-d02e-49b7-9727-d4faf2867b10)

**Comment by moovida** [comment]
Ubuntu 24.04. Same here.

**Comment by boustanihani** [comment]
Same issue here, using NixOS 25.05, Gnome & XORG.

And I found this workaround, if u disable the "AppIndicator" extension and enable it again, then the dropbox tray icon will work.

<img width="613" height="647" alt="Image" src="https://github.com/user-attachments/assets/1a9089a9-4eda-45cd-b8ca-6385d0247039" />

**Comment by josecastillolema** [comment]
Could the issue be that the icons are not being properly exported to **/var/lib/flatpak/exports/share/icons/hicolor** ?

I do see them (locally within the flatpak) in **/app/share/icons**

**Comment by vitaminace33** [comment]
Aside of already experiencing this issue in 2023...

> For me it is working fine on Ubuntu 22.04, but not responding on Manjaro.

I am currently experiencing it on EndeavourOS (on two different machines). However, I use the unofficial distro package from AUR, **not flathub**.

@boustanihani's workaround does work around it :-D

> And I found this workaround, if u disable the "AppIndicator" extension and enable it again, then the dropbox tray icon will work.
